wafer/talks/templates/wafer.talks/talk.html

```diff
@@ -41,6 +41,13 @@
 <div class="well">
     {{ object.abstract.rendered|safe }}
 </div>
+{% if show_private_notes and perms.edit_private_notes and object.private_notes %}
```

Comment:
No worries, @hodgestar. Removing the setting is trivial, but is there a reason why not simply keep it and default it to `True`? Or do you just prefer the code to stay as lean as possible?